pipelines.yml

```diff
@@ -86,6 +96,42 @@ pipelines:
           onComplete:
             - update_commit_status buildJfrogCliPluginGit --context "$step_name"
 
+  - name: upgrade_plugin_on_core_release
+    configuration:
+      runtime:
+        type: image
+        image:
+          custom:
+            name: releases-docker.jfrog.io/jfrog-ecosystem-integration-env
+            tag: 1.0.0
+    steps:
+      - name: upgrade_plugin_on_core_release
+        type: Bash
+        configuration:
+          inputResources:
+            - name: jfrog_cli_core_trigger
+          integrations:
+            - name: ILAutomation2
+            - name: JFrogCLIPluginsRegBot
+          onExecute:
+            - git clone https://github.com/jfrog/jfrog-cli-plugins.git && jfrogCliPluginsPath=`pwd`/jfrog-cli-plugins
+            - git clone https://github.com/jfrog/jfrog-cli-plugins-reg.git && cd jfrog-cli-plugins-reg/pipelinesScripts/validator
+            - env -i PATH=$PATH issue_token=$int_JFrogCLIPluginsRegBot_token HOME=$HOME PWD=$PWD JAVA_HOME=$JAVA_HOME GRADLE_HOME=$GRADLE_HOME MAVEN_HOME=$MAVEN_HOME go run validator.go upgradejfrogplugins $jfrogCliPluginsPath
+            - cd $jfrogCliPluginsPath
+            # Override origin.
+            - git remote set-url origin https://github.com/jfrog/jfrog-cli-plugins.git && git fetch origin main && git checkout main
+            # Check if new commits were added.
+            - "[[ $(git log origin/main...main --oneline | wc -l) -eq 0 ]]"
+            # Check if staged files contains none .mod or .sum files.
```

Comment:
Check if staged files contains none .mod or .sum files.
-->
Check if the staged files do not include .mod or .sum files.